src/runtime_src/core/common/api/command.h

```diff
@@ -64,6 +64,12 @@ class command : public std::enable_shared_from_this<command>
   virtual void
   notify(ert_cmd_state) const = 0;
 
+  /**
+  * get_hw_context() - get hw context of command buffer
+  */
+  virtual xrt::hw_context
+  get_hw_context() const = 0;
+
```

Comment:
should not be needed

Reply:
needed as we are embedding xrt::hw_context in kernel_command.